multimodal/src/autogluon/multimodal/models/fusion.py

```diff
@@ -243,6 +243,8 @@ def __init__(
         head_normalization: Optional[str] = "layer_norm",
         adapt_in_features: Optional[str] = None,
         loss_weight: Optional[float] = None,
+        additive_attention: Optional[bool] = False,
+        share_qv_weights: Optional[bool] = False,
```

Comment:
Need docstrings for `additive_attention` and `share_qv_weights`.

Reply:
added